docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -386,7 +381,20 @@ For `enable`, `disable`, `delete`, `edit`, and `duplicate` actions, a JSON objec
                "throttle":"no_actions",
                "actions":[
 
-               ]
+               ],
+               "execution_summary": {                 <1>
+                 "last_execution": {
+                   "date": "2022-03-23T16:06:12.787Z",
+                   "status": "partial failure",
+                   "status_order": 20,
+                   "message": "This rule is attempting to query data from Elasticsearch indices listed in the \"Index pattern\" section of the rule definition, however no index matching",
```

Comment:
I assume this is just a sample message, but if it's possible to edit the actual message that the API returns, that would be great, too. Also make the same edits to the other APIs in the PR.
```suggestion
                   "message": "This rule is attempting to query data from Elasticsearch indices listed in the \"Index pattern\" section of the rule definition, but no matching index was found.",
```

Reply:
Applied your suggestion, thanks! Will also edit the actual message.